build/Dockerfile

```diff
@@ -7,8 +7,12 @@ RUN curl --silent --location https://deb.nodesource.com/setup_5.x | bash - && \
  apt-get install --no-install-recommends --auto-remove -y git build-essential file nodejs && \
  apt-get clean autoclean && \
  apt-get autoremove -y && \
+ curl --silent --location http://www.musl-libc.org/releases/musl-1.1.12.tar.gz | tar -xz && \
+ cd musl-1.1.12 && ./configure && make && make install && \
  rm -rf /tmp/*
 
+ENV CC=/usr/local/musl/bin/musl-gcc
```

Comment:
It's not strictly necessary to remove the "install musl" bit, but it's wasteful to leave it in.